tchannel/messages/common.py

```diff
@@ -66,6 +67,22 @@
 )
 
 
+def random_tracing():
+    """
+    Create new Tracing() tuple with random IDs.
+    """
+
+    def _uniq_id():  # TODO: cython
+        """Create a random 64-bit unsigned long."""
+        return random.getrandbits(64)
```

Comment:
merely from encapsulation pov, as nobody else needs it. But if it's detrimental to performance, I can move it.